Rmd/DESeq2_report.rnaseq.Rmd

```diff
@@ -936,24 +950,31 @@ for (x in 1:nrow(contrasts)) {	# For all comparisons to be done
 		sampleColsg1 <- grep(dimnames(SampPerGroup)[[1]][1],DE_Design[,DESIGN])
 		sampleColsg2 <- grep(dimnames(SampPerGroup)[[1]][2],DE_Design[,DESIGN])
 		
-	  Check <- median(as.numeric(DECounts[gene, sampleColsg1])) > quantile(DECounts[gene, sampleColsg2], 0.75)[[1]]
+		# Same problem as above, use names explictly
+		sampleNames_g1 <- rownames(DE_Design[sampleColsg1,])
+		sampleNames_g2 <- rownames(DE_Design[sampleColsg2,])
+		
+	  Check <- median(as.numeric(DECounts[gene, sampleNames_g1])) > quantile(DECounts[gene, sampleNames_g2], 0.75)[[1]]
 		quantilePass <- c(quantilePass, Check)
 		
-		Check <- median(as.numeric(DECounts[gene, sampleColsg2])) > quantile(DECounts[gene, sampleColsg1], 0.75)[[1]]
+		Check <- median(as.numeric(DECounts[gene, sampleNames_g2])) > quantile(DECounts[gene, sampleNames_g1], 0.75)[[1]]
 		quantilePass <- c(quantilePass, Check)
 			
-		if (sum(quantilePass) > 0) {
+		if (sum(quantilePass) > 0 ) {
 		  Filter[gene, 2] <- 1
-		}	else {
+		# }	else if (Platform == "TempO-Seq") { # This is a hack... TempOSeq data is messy, so we are ignoring this filter altogether.
+		#   Filter[gene, 2] <- 1
+		} else {
```

Comment:
ditto. Not sure if the commented out lines were code you were using for tests or if you want to keep them in the code

Reply:
Good points, they were for testing before I figured out what was going on with the quantile filtering. Will remove the commented things you noted.